src/seedu/addressbook/data/person/Name.java

```diff
@@ -9,7 +9,7 @@
  * Represents a Person's name in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidName(String)}
  */
-public class Name {
+public class Name implements Comparable<Name> {
```

Comment:
I personally prefer this implementation (making `name` a `comparable` object) so 👍